### ISSUE TITLE
Add Community responders to the side panel - CHW

### ISSUE THREAD

**Issue by msschroeder** [opensrp/opensrp-client-chw-emergency-transport#7]
- [x] Add "Community responders" to the side panel (see the image below)
- [x] @rowo Can you please provide an icon that would be appropriate for this?
- [x] I should be in the settings part of the side panel. It should be above the selected language and below the Stock usage.
- [x] When you click, it will take you to the list of the community responders (https://github.com/OpenSRP/opensrp-client-chw-emergency-transport/issues/6 )

![Screenshot_20200326-083405](https://user-images.githubusercontent.com/20777928/77663263-ef0e8700-6f8d-11ea-8dd6-4831978e4f9a.png)

**Comment by githengi** [comment]
LOE 1 days for 1 FTE

**Comment by rowo** [comment]
@msschroeder just to be clear, the icon should represent emergency transportation people to call? And this could be a an ambulance, taxi or even a tuk tuk?

**Comment by msschroeder** [comment]
@rowo Yes, this would be the emergency transport people. The ambulance would probably be the most descriptive

**Comment by rowo** [comment]
@msschroeder 
![icon-emergency-transport](https://user-images.githubusercontent.com/1584163/78743697-57b21800-792d-11ea-9a52-c48f1cca60d3.png)
<img src="https://user-images.githubusercontent.com/1584163/78743702-5aad0880-792d-11ea-8148-c6b34a76f722.png" width="280">

**Comment by msschroeder** [comment]
@rowo That looks awesome

**Comment by Mstjamush** [comment]
@msschroeder  should it be "Emergency transport" or "Community responders". It's explained as "Community responders" but appears as "Emergency transport" in one of the designs.

**Comment by msschroeder** [comment]
@Mstjamush It should be "Community responders" because it is just the register of forms for the community responders not the emergency transport module
